src/components/converter/Form.js

```diff
@@ -25,6 +25,8 @@ const options = ['ANT', 'DAI', 'ETH', 'USDC']
 const ANJ_MIN_REQUIRED = bigNum(10)
   .pow(18)
   .mul(10000)
+const SLIPPAGE_LOWER_BOUND = bigNum(toWei('9900'))
+const SLIPPAGE_UPPER_BOUND = bigNum(toWei('10100'))
```

Comment:
Right now we basically have a window of ~200 ANJ for slippage. Should this be even lower?

Reply:
We don't need the slippage upper bound; as we only really care about the 10k ANJ requirement (transactions trying to activate less than that will fail).

For the minimum though, it's odd we use 1% here, but 5% in the actual conversion logic. When I see these cases, I would lift this up to a utility function and add a constant so it's easy to adjust what min slippage value we want as we see users interacting (or failing to interact) with the contract.